src/main/java/pixel/Parser.java

```diff
@@ -0,0 +1,80 @@
+package pixel;
+
+/**
+ * Class that deals with the control flow upon obtaining user command
+ */
+public class Parser {
+
+    /**
+     * Constructor method of Parser
+     */
+    public Parser() {
+    }
+
+    /**
+     * Act based on command given from the user's input
+     *
+     * @param command command obtained from Ui
+     * @param tasks TaskList for processing of tasks
+     * @param storage Storage for processing of files
+     */
+    public static void parse(String command, TaskList tasks, Storage storage) {
```

Comment:
Perhaps use a switch here instead of multiple if and else if statements? It would make the code more readable. 😄